### ISSUE TITLE
Cannot install Pillow 8.2.0 via pip on FreeBSD

### ISSUE THREAD

**Issue by frozenpandaman** [python-pillow/Pillow#5559]
I can't install Pillow 8.2.0 via pip on FreeBSD 12.2. Installing 8.1.2 in the same manner still works fine.

In a new, fresh virtualenv, running `pip install pillow` gives the following (extremely long) error:
https://pastebin.com/5QeRLjb9

The error ends with "error: command 'cc' failed with exit status 1" and "ERROR: Command errored out with exit status 1" (in case this helps anyone Googling it in the future…)

Installing `wheel` (v0.36.2, to avoid the "Using legacy 'setup.py install' for pillow, since package 'wheel' is not installed." warning) doesn't solve it.

I'm using Python 3.8.9 and pip 21.1.3.

Thanks!

**Comment by radarhere** [comment]
If you're interested, you could try installing it from a package rather than through pip - https://www.freebsd.org/cgi/ports.cgi?query=py38-pillow&stype=all&sektion=all

**Comment by frozenpandaman** [comment]
Thanks! I saw [this section](https://pillow.readthedocs.io/en/stable/installation.html#freebsd-installation) of the docs too and unfortunately it seems my shared hosting doesn't give me permission to install packages via `pkg`:

```
$ pkg install py38-pillow-8.2.0
pkg: Insufficient privileges to install packages
```

**Comment by nulano** [comment]
Searching for "error" in the provided output shows the cause is:
```
  src/_imagingft.c:280:60: error: token is not a valid binary operator in a preprocessor subexpression
  #if !defined(RAQM_VERSION_ATLEAST) || !RAQM_VERSION_ATLEAST(0, 7, 0)
                                        ~~~~~~~~~~~~~~~~~~~~~^
  1 error generated.
```
This should be solved by #5462 in Pillow 8.3.0 due to release on July 1.

You can also try updating Raqm to 0.7.0 or later, or disabling it with `pip install Pillow --global-option build_ext --global-option --disable-raqm`.

**Comment by frozenpandaman** [comment]
Thanks a ton! That worked, and glad to know it'll be fixed in 8.3.0 releasing very soon. :) I'll close this for now, then.
